.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Go ahead and add the following to this file:
```
AllCops:
  DisplayCopNames: true

Layout/EmptyLinesAroundModuleBody:
  Enabled: false

Metrics/ModuleLength:
  Enabled: false

Metrics/MethodLength:
  Enabled: false
```

This will help remove some of the cops that are, in my opinion, more of a nuisance.